client/interactive.js

```diff
@@ -53,13 +51,13 @@ export class FormulaBunBotInteracive extends FormulaBunBotBase {
     if (!this.readyAt) return;
     this.server.serverinfo = serverinfo;
     const stat = `${serverinfo.numberofplayers} players race`;
-    this.user
-      .setActivity(stat, { type: "WATCHING" })
-      .then(() => {
-        if (serverinfo.numberofplayers === 0) this.user.setStatus("idle");
-        else this.user.setStatus("online");
-      })
-      .catch(console.error);
+    try {
+      this.user.setActivity(stat, { type: 'WATCHING' });
+      if (serverinfo.numberofplayers === 0) this.user.setStatus('idle');
+      else this.user.setStatus('online');
+    } catch (err) {
+      console.error(err);
+    }
```

Comment:
hmm, can't remember if it was required for `setStatus` to happen after `setActivity`. Needs testing ig.

Reply:
I did this because apparently `setActivity` doesn't return a promise anymore.